tests/transforms/test_insert_ops.py

```diff
@@ -51,10 +51,11 @@ def test_multiwire_op(self):
         with pytest.raises(ValueError, match="Only single-qubit operations can be inserted into"):
             insert(qml.CNOT, [])(self.tape)
 
-    def test_invalid_position(self):
+    @pytest.mark.parametrize("pos", [1, ["all", qml.RY, int], "ABC", str])
+    def test_invalid_position(self, pos):
         """Test if a ValueError is raised when an invalid position is requested"""
         with pytest.raises(ValueError, match="Position must be either 'start', 'end', or 'all'"):
```

Comment:
Shouldn't this line change to match the new error message raised by the ValueError (line 222 - 225)

Reply:
`match` is regex-based, and the beginning is the same for both old and new error messages. Thus, I left it as is.